src/syrupy/serializers/amber.py

```diff
@@ -89,7 +92,9 @@ def object_type(cls, data: "SerializableData") -> str:
         return f"<class '{data.__class__.__name__}'>"
 
     @classmethod
-    def serialize_string(cls, data: "SerializableData", indent: int = 0) -> str:
+    def serialize_string(
+        cls, data: "SerializableData", *, indent: int = 0, visited: List[Any] = []
```

Comment:
we can remove the unused `visited` kwarg if we do `**_`, not sure how mypy handles that though?

Reply:
I think it complained last I tried, will attempt again but treating as non blocking

Reply:
Too much refactoring needed to satisfy mypy and ignore the argument, leaving as is